app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ErrorsController < ApplicationController
+  def not_found; end
```

Comment:
as you know, now the http status code will be 200 which is not good.
so change it as: 
```suggestion
  def not_found
    render status: 404
   end
```